gradle.properties

```diff
@@ -49,7 +49,7 @@ buildId = 0
 productName = Apache Geode
 productOrg = Apache Software Foundation (ASF)
 
-minimumGradleVersion = 5.4
+minimumGradleVersion = 5.5.1
```

Comment:
Why the bump to Gradle 5.5.1 if we aren't also moving to the latest JMH plugin?

Reply:
I started to update the plugin here but then solved it another way. I can back out this Gradle upgrade here and open another PR.